include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1615,7 +1632,9 @@ MGTwoLevelTransfer<dim, LinearAlgebra::distributed::Vector<Number>>::prolongate(
   LinearAlgebra::distributed::Vector<Number> &      dst,
   const LinearAlgebra::distributed::Vector<Number> &src) const
 {
-  const unsigned int vec_size = VectorizedArray<Number>::n_array_elements;
+  using VectorizedArrayType = VectorizedArray<Number>;
+
+  const unsigned int vec_size = VectorizedArrayType::n_array_elements;
```

Comment:
```suggestion
  const unsigned int n_lanes = VectorizedArrayType::size();
```
I think that is the common name in other files.

Reply:
I have written these lines of code (to be precise: copied these lines) far before with consistently renamed this alias in the `matrix_free` folder ;)

I used the opportunity to rename the alias also in `MGTransferMatrixFree` !

Reply:
Ready from my side after I have rebased with #10966.